### PR TITLE
feat: Opening and closing portal switch

### PR DESCRIPTION
**Pull request by iamsivin**
# Pull Request Template

## Description

This PR includes,

1. Popover switch flow (opening and closing)
2. Fix the i18n issue.
3. Moved sidebar and header components to new folder path.

Fixes https://github.com/chatwoot/product/issues/484

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How Has This Been Tested?

**Screencast**

https://user-images.githubusercontent.com/64252451/181213823-330a0954-1233-43b1-b877-bd3278f445c6.mov

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by muhsin-k — DISMISSED** [review]

File: app/javascript/dashboard/routes/dashboard/helpcenter/components/HelpCenterLayout.vue [hunk 1]
```diff
@@ -171,6 +180,138 @@ export default {
         },
       ];
     },
+    portals() {
```
Comment (fayazara) [inline app/javascript/dashboard/routes/dashboard/helpcenter/components/HelpCenterLayout.vue:183]: Maybe this sample portal data can be added in the stories file

**Review by fayazara — CHANGES_REQUESTED** [review]
Looks good! Have left a small comment

File: app/javascript/dashboard/routes/dashboard/helpcenter/components/HelpCenterLayout.vue (hunk 1 above)
Comment (iamsivin) [reply app/javascript/dashboard/routes/dashboard/helpcenter/components/HelpCenterLayout.vue:183]: @fayazara Done https://github.com/chatwoot/chatwoot/pull/5126/commits/0231ff8baec0b0dbca9e880d80a0400427bf7fc0

**Review by muhsin-k — APPROVED** [review]

**Review by fayazara — APPROVED** [review]
Looks good!

**Pull request merged**